apps/trading/client-pages/settings/settings.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from '@vegaprotocol/i18n';
+import { TelemetryApproval } from '../../components/welcome-dialog/telemetry-approval';
+import {
+  Divider,
+  RoundedWrapper,
+  ThemeSwitcher,
+} from '@vegaprotocol/ui-toolkit';
+
+export const Settings = () => {
+  return (
+    <div className="py-4 px-8 flex w-full justify-center">
+      <div className="min-w-[300px] ">
+        <div className="text-3xl uppercase font-alpha liga calt">
```

Comment:
spacing has been improved

Reply:
@macqbat can we
- use a correct html element (h1)
- bump the the font-size to text-5xl

@joellecharlotte text-5xl this isn't exactly 54px but out of the current available sizes configured, this is the closest. I'll make another ticket to adjust our config so that we can get it bang on. Note that this will affect other headers in governance that already use text-5xl

Reply:
done, I adjusted also some margins and other sizes